en_US/access-control/authn/redis.md

```diff
@@ -45,37 +45,37 @@ So, we can also configure `cmd` as `HMGET mqtt:${username} password salt is_supe
 - **Password** (optional): Specify Redis user password. 
 
 **TLS Configuration**: Turn on the toggle switch if you want to enable TLS. For more information on enabling TLS, see [Network and TLS](../../network/overview.md).
 
 **Connection Configuration**: Set the concurrent connections.
 
-- **Pool size** (optional): Input an integer value to define the number of concurrent connections from an EMQX node to a Redis server. Default: **8**. 
+- **Pool size** (optional): Specify the number of concurrent connections from an EMQX node to a Redis server. Default: `8`. 
 
-**Authentication configuration**: Fill in the authentication-related settings:
+**Authentication configuration**: Configure settings related to authentication:
 
-- **Password Hash**: Select the Hash function for storing the password in the database, for example, plain, md5, sha, bcrypt, pbkdf2. 
-  - If **plain**, **md5**, **sha**, **sha256** or **sha512** are selected, we also need to configure:
-    - **Salt Position**: Specify the way (**suffix**, **prefix**, or **disable**) to add salt (random data) to the password. You can keep the default value unless you are migrating user credentials from external storage into EMQX built-in database. Note: If **plain** is selected, the **Salt Position** should be **disable**. 
-  - If **bcrypt** is selected, you also need to configure:
-    - **Salt Rounds**: Specify the calculation times of Hush function (2^Salt Rounds). Default value: **10**; Value range **4~31**. You are recommended to use a higher value for better protection. Note: Increasing the cost factor by 1 doubles the necessary time. 
-  - If **pkbdf2** is selected, we also need to configure:
-    - **Pseudorandom Function**: Specify the Hush functions to generate the key, such as sha256. 
-    - **Iteration Count**: Specify the iteration times; Default: 4096
-    - **Derived Key Length** (optional): Specify the length of the generated password. You can leave this field blank, then the key length will be determined by the pseudorandom function you selected. 
+- **Password Hash**: the hash function for storing passwords in the database. Options include `plain`, `md5`, `sha`, `bcrypt`, and `pbkdf2`. Additional configuration depends on your selected function:
+  - For `plain`, `md5`, `sha`, `sha256`, or `sha512`:
+    - **Salt Position**: Determine how salt (random data) is added to the password. Options are `suffix`, `prefix`, or `disable`. Keep the default value unless you migrate user credentials from external storage into the EMQX built-in database. Note: Set to `disable` if `plain` is selected. 
+  - For `bcrypt`:
+    - **Salt Rounds**: Set the number of applications of the hash function, expressed as 2^Salt Rounds, also known as the "cost factor". The default is `10`, with a range of `5` to `10`. Opting for a higher value enhances security. Note: Increasing the cost factor by 1 doubles the necessary time for authentication.
```

Comment:
In the latest schema, it looks like the Salt Rounds configuration no longer exists
<img width="911" alt="image" src="https://github.com/emqx/emqx-docs/assets/25006774/acfd3df7-d369-4736-a11a-4b784d8309ca">

Reply:
`bcrypt` is an auxiliary schema for the two really uses: `bcrypt_rw` and `bcrypt_rw_api`

Reply:
<img width="917" alt="image" src="https://github.com/emqx/emqx-docs/assets/25006774/f0ec0bc8-5707-4e5b-bfb6-54f3f1500b1a">
I see that the Salt Rounds configuration is also missing from the dashboard, was it removed (for simplicity of configuration)?
If this field is not present on dashbaord, we don't need to inform users about it in the documentation.